notebooks/rbm_training.ipynb

```diff
@@ -0,0 +1,638 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        for i in range(steps):
</pre><p>Maybe replace for <a href="https://jax.readthedocs.io/en/latest/_autosummary/jax.lax.scan.html" target="_blank">jax.lax.scan</a> or the <a href="https://jax.readthedocs.io/en/latest/jax.experimental.loops.html" target="_blank">loops module</a> in Jax</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/probml-notebooks/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>